src/main/java/com/noticemedan/mappr/viewmodel/CanvasView.java

```diff
@@ -398,17 +398,22 @@ public void toggleReversedBorders() {
 		this.showReversedBorders = !this.showReversedBorders;
 	}
 
-	private void drawPoi() {
+	private void drawPointer() {
 		if (this.pointerPosition == null) return;
-		double size = this.viewRect.getWidth() * 0.0001;
-		double width = pointer.getWidth() * size;
-		double height = pointer.getHeight() * size;
+		drawImage(this.pointer,this.pointerPosition,0.00008,false);
+	}
+
+	private void drawImage(BufferedImage img, Point2D coordinate, double size, boolean center) {
+		double scaling = (this.zoomLevel < 100) ? this.viewRect.getWidth() * size :	0.01 * size;
```

Comment:
Lille ting: Kunne man på en eller anden måde gøre pointerens størrelse uafhængig af viewRect getWidth? Så pointeren ikke ændre størrelse når man hiver i vinduet?

Reply:
Nej, for så ville den fylde hele skærmen når man zoomer ind. Måske man kan lave det om til at være afhængigt af zoomlevel istedet, men kommer til at tage en del tid at finde ud af de rigtige værdier at sætte størrelsen til, hvilket jeg synes er lidt spild af tid giver vi skal aflevere på tirsdag. Men jeg synes du skal prøve den nye pointer før du sætter for høj prioritet på det. Den er blevet mindre og fylder ikke nær så meget som den anden, heller ikke i fullscreen :)

Reply:
Men jeg skal nok love at se om det kan lade sig gøre med et hurtigt fix i en af mine pauser.